helpers/db.json

```
@@ -0,0 +1,66 @@
+[
+  {
+    "id": 1,
+    "article": "Body",
```

I recommend to rename this key 👇

```suggestion
    "name": "Body",
```